specification/immunisation-fhir-api.yaml

```diff
@@ -185,7 +263,7 @@ paths:
               properties: { }
       responses:
         "200":
-          description: Search Immunization operation successful
+          description: Search Immunisation operation successful
```

Comment:
Why capital? It should be either "Immunization" (capital and z) or "immunisation" (lower and s).